fc-community-contracts/src/FactChainNFT.sol

```diff
@@ -0,0 +1,59 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.20;
+
+import {ERC721URIStorage} from "./ERC721/extensions/ERC721URIStorage.sol";
+import {ERC721} from "./ERC721/ERC721.sol";
+import {Ownable} from "./utils/Ownable.sol";
+
+/// @title FactChainers Community Notes NFT
+/// @author Pierre HAY
+/// @notice
+/// @dev
+contract FactChainNFT is Ownable, ERC721URIStorage {
+    uint256 public tokenIdCounter;
+    string private baseTokenURI;
+
+    constructor(address _owner, string memory _name, string memory _symbol, string memory _baseTokenURI)
```

Comment:
I think we can hardcode a `name` and `symbol` since our goal is not to deploy many 721 collections.